src/storage-blob-preview/azext_storage_blob_preview/tests/latest/test_storage_blob_live_scenarios.py

```diff
@@ -45,14 +45,20 @@ def test_storage_blob_upload_10G_file(self, resource_group, storage_account):
         self.verify_blob_upload_and_download(resource_group, storage_account, 10 * 1024 * 1024,
                                              'block', skip_download=True)
 
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer()
+    def test_storage_blob_upload_5G_file_with_fixed_block_size(self, resource_group, storage_account):
```

Comment:
with previous api version, we already have the capability to upload 10GB blob. How can we test the new api version support jumbo blob then?

Reply:
See https://github.com/Azure/azure-cli-extensions/pull/2867#discussion_r584463312